internal/pkg/broker/broker_utils.go

```diff
@@ -0,0 +1,130 @@
+package broker
+
+import (
+	"context"
+	"net/http"
+	"strconv"
+
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/kafkarest"
+	"github.com/confluentinc/kafka-rest-sdk-go/kafkarestv3"
+	"github.com/spf13/cobra"
+)
+
+const AbbreviationLength = 25
```

Comment:
Abbreviation length of what? The broker ID? The broker name?

Reply:
when describing broker configs, the abbreved version of config name (e.g. ssl.trustmanager.algorith...